subsys/esb/esb_dppi.c

```diff
@@ -40,8 +42,17 @@ void esb_ppi_for_txrx_set(bool rx, bool timer_start)
 	nrf_dppi_channels_include_in_group(ESB_DPPIC, BIT(egu_ramp_up), ramp_up_dppi_group);
 
 	nrf_egu_subscribe_set(ESB_EGU, ESB_EGU_DPPI_TASK, egu_timer_start);
-	nrf_radio_subscribe_set(NRF_RADIO, rx ? NRF_RADIO_TASK_RXEN : NRF_RADIO_TASK_TXEN,
-				egu_ramp_up);
+	if (state == NRF_RADIO_STATE_TXIDLE || state == NRF_RADIO_STATE_TX) {
+		/* When CONFIG_ESB_KEEP_TX_IDLE is enabled the radio is running,
+		 * so we need to trigger NRF_RADIO_TASK_START.
```

Comment:
```suggestion
		/* When CONFIG_ESB_KEEP_TX_IDLE is enabled, the radio is running.
		 * NRF_RADIO_TASK_START needs to be triggered.
```
Not really our cup of tea to fix the comments in the code. However decided to leave a minor suggestion.
It is not a good idea to talk about "we".

Reply:
Thanks for your comment.